core/src/main/java/com/linecorp/armeria/client/retry/ExponentialBackoff.java

```diff
@@ -95,4 +95,33 @@ public String toString() {
                           .add("multiplier", multiplier)
                           .toString();
     }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
```

Comment:
Code style) We prefer top-level builder classes instead of nested classes.
Could you move this class to `ExpontionalBackoffBuilder`?